### DISCUSSION TITLE
react class name quick fix

### DISCUSSION THREAD

**Discussion by ianzone** [biomejs/biome#2501] [Ideas]
<img width="422" alt="image" src="https://github.com/biomejs/biome/assets/20043230/e03cb9f1-7615-490b-90ae-540589925b13">

Hope Biome can do auto quick fix for this

<img width="439" alt="image" src="https://github.com/biomejs/biome/assets/20043230/3bbce719-ddb3-4c8f-95ea-1f0266367c7b">

**Comment by arendjr** [comment]
I’m afraid this request is out of scope for Biome. I would expect something like this to be implemented in the IDE instead, possibly as an extension. Not sure if VS Code snippets can also help with this kind of thing?
